tests/DotNetBoilerplate.Tests.Integration/Setup/BookstoreEndpointsTestsFixture.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.AspNetCore.TestHost;
+using Npgsql;
+using Respawn;
+using Xunit;
+
+namespace DotNetBoilerplate.Tests.Integration.Setup;
+
+public sealed class BookstoreEndpointsTestsFixture : IAsyncLifetime
```

Comment:
Tu już moja sprawka bo pisaliśmy sobie na szybko w celach demonstracyjnych xd Nazwijcie to proszę BookstoreIntegrationTestsFixture